<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Index</class>
 <widget class="QWidget" name="Index">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>天堂证券中心</string>
  </property>
  <widget class="QFrame" name="frame_window">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>736</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame[objectName=&quot;frame_window&quot;]
{
	border-radius:15px;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>695</x>
      <y>2</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	border-top-right-radius:14px;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(255, 0, 0);
}

</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>655</x>
      <y>2</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	color:white;
	font: 20pt &quot;Calibri&quot;;/*字体*/
	background-color: rgba(255, 255, 255,0);/*背景色*/
	border:0px;
}
QPushButton:hover
{
	background-color: rgba(255, 255, 255,0.5);
	color: rgb(0, 0, 0)
}

</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>2</y>
      <width>732</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;
color:rgb(20,20,20);
padding-bottom:1px;
border-top-left-radius:14px;
border-top-right-radius:14px;
background-color: rgb(110, 205, 255)</string>
    </property>
    <property name="text">
     <string>天堂证券中心</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListView" name="lv_members">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>75</y>
      <width>161</width>
      <height>396</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QListView
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(250, 250, 250);
	outline:0px;
}

QListView::item
{
	height:25px;
	border-radius:10px;
}

QListView::item:hover
{
	background-color: rgb(157, 213, 255);
	height:25px;
	border-radius:10px;
}

QListView::item:selected
{
	background-color: rgb(85, 170, 255);
	border-radius:10px;
	color:black;
}</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_search">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>40</y>
      <width>125</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:0px solid gray;
border-radius:7px;
padding-left:3px;
padding-bottom:2px;
font:13px;
background-color: rgb(250, 250, 250);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>270</y>
      <width>536</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string>折线图（伪）</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_add_user">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>新增用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_remove_user">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>删除用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edit_user">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>修改用户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_freeze_user">
    <property name="geometry">
     <rect>
      <x>495</x>
      <y>210</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
	color:red;
}
QPushButton::hover
{
	border:2px solid red;
}
</string>
    </property>
    <property name="text">
     <string>冻结账户</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_unfreeze_user">
    <property name="geometry">
     <rect>
      <x>605</x>
      <y>210</y>
      <width>71</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
	border:0px solid gray;
	border-radius:10px;
	background-color: rgb(232, 232, 232);
}
QPushButton::hover
{
	border:2px solid rgb(0, 170, 255);
}
</string>
    </property>
    <property name="text">
     <string>解冻账户</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="de_birthday">
    <property name="geometry">
     <rect>
      <x>345</x>
      <y>215</y>
      <width>116</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255,0);
border:1px solid white;
border-left-color: rgba(255, 255, 255,0);
border-right-color: rgba(255, 255, 255,0);
border-top-color: rgba(255, 255, 255,0);
border-bottom-color: rgb(85, 170, 255);
font: 15pt &quot;Calibri&quot;;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_male">
    <property name="geometry">
     <rect>
      <x>355</x>
      <y>155</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13px</string>
    </property>
    <property name="text">
     <string>男</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rb_female">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>155</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:13px</string>
    </property>
    <property name="text">
     <string>女</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_name">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>95</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	
	background-color: rgba(255, 255, 255,0);
	border:1px solid white;
	border-left-color: rgba(255, 255, 255,0);
	border-right-color: rgba(255, 255, 255,0);
	border-top-color: rgba(255, 255, 255,0);
	border-bottom-color: rgb(85, 170, 255);
	font: 15pt &quot;Calibri&quot;;
}

QLineEdit:hover
{
	border-bottom-color: rgba(85, 170, 255,0.6);
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_state_value">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>155</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;
font:15px;</string>
    </property>
    <property name="text">
     <string>正常</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_money_value">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>75</y>
      <width>181</width>
      <height>66</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:36px;
color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_money">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>100</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>存款：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>515</x>
      <y>88</y>
      <width>31</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:25px;
</string>
    </property>
    <property name="text">
     <string>￥</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_state">
    <property name="geometry">
     <rect>
      <x>485</x>
      <y>155</y>
      <width>76</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>账户状态：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_name">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>100</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_birthday">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>215</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>生日：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sex">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>155</y>
      <width>66</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:17px;</string>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <zorder>lbl_title</zorder>
   <zorder>btn_close</zorder>
   <zorder>btn_minimize</zorder>
   <zorder>lv_members</zorder>
   <zorder>le_search</zorder>
   <zorder>label</zorder>
   <zorder>btn_add_user</zorder>
   <zorder>btn_remove_user</zorder>
   <zorder>btn_edit_user</zorder>
   <zorder>btn_freeze_user</zorder>
   <zorder>btn_unfreeze_user</zorder>
   <zorder>de_birthday</zorder>
   <zorder>rb_male</zorder>
   <zorder>rb_female</zorder>
   <zorder>le_name</zorder>
   <zorder>lbl_state_value</zorder>
   <zorder>lbl_money_value</zorder>
   <zorder>lbl_money</zorder>
   <zorder>label_5</zorder>
   <zorder>lbl_state</zorder>
   <zorder>lbl_name</zorder>
   <zorder>lbl_birthday</zorder>
   <zorder>lbl_sex</zorder>
  </widget>
  <widget class="QFrame" name="frame_blur">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>736</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgba(255, 255, 255,0.9);
border:2px solid gray;
border-radius:15px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_search">
   <property name="geometry">
    <rect>
     <x>18</x>
     <y>48</y>
     <width>23</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image:url(Resources/find_25x25.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_welcome">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>45</y>
     <width>571</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(85, 255, 255,0);
border:0px;
font: 13px;</string>
   </property>
   <property name="text">
    <string>欢迎！xxx   用户类型：管理员</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_state_div">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>75</y>
     <width>536</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(85, 255, 255,0);
border:1px solid gray;
font: 13px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>frame_blur</zorder>
  <zorder>frame_window</zorder>
  <zorder>lbl_search</zorder>
  <zorder>lbl_welcome</zorder>
  <zorder>lbl_state_div</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btn_minimize</sender>
   <signal>clicked()</signal>
   <receiver>Index</receiver>
   <slot>btn_minimize_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>735</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>712</x>
     <y>-15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Index</receiver>
   <slot>btn_close_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>653</x>
     <y>-6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btn_minimize_click()</slot>
  <slot>btn_close_click()</slot>
  <slot>btn_get_money_click()</slot>
  <slot>btn_save_money_click()</slot>
 </slots>
</ui>
